src/EasyAbp.Abp.EventBus.Boxes.Dtm.Http/EasyAbp.Abp.EventBus.Boxes.Dtm.Http.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net6.0</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <ProjectReference Include="..\EasyAbp.Abp.EventBus.Boxes.Dtm\EasyAbp.Abp.EventBus.Boxes.Dtm.csproj" />
+    </ItemGroup>
+
+    <ItemGroup>
+      <PackageReference Include="Dtmcli" Version="1.2.0" />
```

Comment:
Rename `DtmgrpcVersion` in "Directory.Build.props" to `DtmSdkVersion`. Keep using version 1.1.2 (don't upgrade to 1.2.0 since it's not related to this PR)

Use `$(DtmSdkVersion)` here.